packages/experimental/RadioGroup/src/RadioGroup/useRadioGroup.ts

```diff
@@ -35,6 +35,7 @@ export const useRadioGroup = (props: RadioGroupProps): RadioGroupInfo => {
       ...props,
       required,
       layout,
+      accessible: true,
```

Comment:
Does this fix a bug?

I think it's worth mentioning in the PR description :)

Reply:
Updated, thanks!